Octokit.Tests.Integration/Clients/RepositoryCommentsClientTests.cs

```diff
@@ -367,4 +367,62 @@ public void Dispose()
             _context.Dispose();
         }
     }
+    public class TheReactionMethod : IDisposable
```

Comment:
the name of the test class should reflect the name of the method.  In this case the method is called `CreateReaction` so the test class should be called `TheCreateReactionMethod`